lz_header.S

```diff
@@ -71,13 +71,15 @@ _entry:
 	 * done with an %ss-relative write.  Therefore, we store the base
 	 * address in %ebp.
 	 *
-	 * Other bits of code also use %ebx and %esi for the image base.  This
+	 * Other bits of code also use %ebx for the image base.  This
 	 * will be cleaned up in due course.
 	 */
 	movl	%eax, %ebx
-	movl	%eax, %esi
 	movl	%eax, %ebp
 
+	/* Set up the Stage 1 stack. */
+	leal	LZ_FIRST_STAGE_STACK_SIZE(%ebp), %esp
```

Comment:
This should probably be `LZ_FIRST_STAGE_STACK_SIZE`, not `_SIZE`

Reply:
Yes.  Fixed.